source/rpt/rpt_calculating_parameters.cc

```diff
@@ -6,6 +6,12 @@
 void
 RPTCalculatingParameters::declare(ParameterHandler &prm)
 {
+  prm.declare_entry("print parameters",
+                    "none",
+                    Patterns::Selection("none|only changed|all"),
```

Comment:
i don't know if only changed or changed is the best choice of a keyword here, that would be my only question. Otherwise dope PR

Reply:
I do not have any preference. I called it "only changed" because in the parameter handler the option is called "KeepOnlyChanged" but if you prefer "changed" that would be also possible

Reply:
Then let's keep only changed if that is coherent with the documentation